src/components/Select/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const Select = ({
+    name,
+    label,
+    options,
+    onChange
+}) => (
+    <div>
+        <label htmlFor={name} />
+        <select id={ name } name={name} onChange={ onChange }>
+            <option value=''>Select one</option>
+            {options.map(({value, text, key}) => <option key={ key } value={ value } >{ text }</option>)}
```

Comment:
Sorry but I don't see the diference here. `{<space>options... )<space>}` is valid though. I'll change it to that